### PR TITLE
Option 2: Remove subspace in favor of ForkedAnsible

### PR DESCRIPTION
**Pull request by steve-gregory**
In this world, we would use [a forked ansible with newly inserted code.](https://github.com/cyverse/ansible/pull/1)

### CODE REVIEW

**Comment by c-mart** [comment]
Spoke with @steve-gregory about this plan last week, and I'm on board with it:
- For now, use [forked Ansible](https://github.com/cyverse/ansible/pull/1) with our changes to provide additional functionality. Stop using custom Subspace middleware.
- Make a good faith effort to contribute these enhancements upstream to Ansible

File: service/deploy.py [hunk 1]
```diff
@@ -339,75 +357,77 @@ def user_deploy_install(instance_ip, username, instance_id, install_action, inst
...
+    directories.sort(cmp=operator.lt)
+    for d in directories:
+        a_dir = d[0]
+        files_in_dir = d[2]
+        files_in_dir.sort()
+        if os.path.isdir(a_dir):
+            for f in files_in_dir:
+                if os.path.splitext(f)[1] == ".yml":
+                    files.append(os.path.join(a_dir, f))
+    return files
```
Comment (cdosborn) [inline service/deploy.py]: A lot of this code could be removed if we leveraged the fact that you can create playbooks which contain other playbooks. We shouldn't be sorting by name. Could create a playbook instance_deploy.yml that includes the playbooks it needs.

Comment (steve-gregory) [reply service/deploy.py]: Interesting idea @cdosborn - Let's get this in _first_ and then make that change separately with a corresponding PR in `atmosphere-ansible`?

**Comment by steve-gregory** [comment]
I'm going to make an effort to get this merged in on Friday, but i'd like to work with @cdosborn or one of the other back-end developers to ensure that `cyverse-ansible` can be signed/packaged without issue.

**Comment by cdosborn** [comment]
Closed in favor of https://github.com/cyverse/atmosphere/pull/631

**Pull request closed without merging**